staging_service/import_specifications/file_writers.py

```diff
@@ -181,6 +185,56 @@ def _check_write_args(folder: Path, types: dict[str, dict[str, list[Any]]]):
     _check_import_specification(types)
 
 
+def write_excel(folder: Path, types: dict[str, dict[str, list[Any]]]) -> dict[str, Path]:
+    """
+    Writes import specifications to an Excel files. All the writers in this module
+    have the same function signatures; see the module level documentation.
+    """
+    _check_write_args(folder, types)
+    res = {}
+    file_ = _IMPORT_SPEC_FILE_NAME + "." + _EXT_EXCEL
```

Comment:
All 3 comments have been addressed locally; they'll show up in a future PR. Renamed `file_` to `filename` and inlined `out` since there was only one occurance.